### PR TITLE
Jenliz3

### PR DESCRIPTION
**Pull request by jenniferbacon01**
Added button on posts page that takes you to a user profile page which currently just has the message Hello. Button is only there when you're signed in.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,6 +28,7 @@ gem "jbuilder", "~> 2.5"
 
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
+gem "launchy"
```
Comment (neoeno) [inline Gemfile]: This feels like a test gem to me

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class UsersController < ApplicationController
+  def show
+    @user = current_user
+    # @posts_sorted = Post.all.sort_by { |post| post.created_at }.reverse
```
Comment (neoeno) [inline app/controllers/users_controller.rb]: Commented out code — I'd suggest deleting

File: spec/features/user_has_profile_spec.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      expect(page).to have_selector('button[type=submit][value="Profile"]')
+    end
+
+    scenario "user clicks profile button and is taken to profile page" do
+      click_link("Profile")
+      expect(page).to have_content('Hello')
+    end
+  end
+
+end
```
Comment (neoeno) [inline spec/features/user_has_profile_spec.rb:34]: Nice test here!

**Review by neoeno — DISMISSED** [review]

**Review by LizzieHard — APPROVED** [review]
Updated from pull request review comments

**Pull request merged**